deploy/ansible/worker/include/run_ursula.yml

```diff
@@ -3,22 +3,135 @@
+        state: directory
+
+    - name: "ensure known nodes directory"
+      become: yes
+      file:
+        path: /home/nucypher/nucypher/known_nodes/metadata
+        state: directory
+
+    - name: Find my public ip
+      uri:
+        url: http://ifconfig.me/ip
+        return_content: yes
+      register: ip_response
+
+    - name: "update Ursula worker config"
+      become: yes
+      become_user: nucypher
+      command: "docker run -v /home/nucypher:/root/.local/share/ -e NUCYPHER_KEYRING_PASSWORD -it {{ nucypher_image | default('nucypher/nucypher:latest') }}  nucypher ursula config --provider {{ blockchain_provider }} --worker-address {{active_account.stdout}} --rest-host {{ip_response.content}} --network {{network_name}} {{nucypher_ursula_init_options | default('')}} --config-file /root/.local/share/nucypher/ursula.json"
+      environment:
+        NUCYPHER_KEYRING_PASSWORD: "{{NUCYPHER_KEYRING_PASSWORD}}"
+
+    - name: "Run Staked Ursula (seed node)"
+      become: yes
+      become_user: nucypher
+      when: SEED_NODE_URI is not undefined and inventory_hostname == SEED_NODE_URI
+      docker_container:
+        recreate: yes
+        name: ursula
+        state: started
+        pull: yes
+        log_driver: json-file
+        log_options:
+          max-size: 10m
+          max-file: "5"
+        image: "{{ nucypher_image | default('nucypher/nucypher:latest') }}"
+        restart_policy: "unless-stopped"
+        command: "nucypher ursula run {{nucypher_ursula_run_options | default('')}} {{signer_options}} --lonely --prometheus --metrics-port 9101"
+        volumes:
+          - /home/nucypher:/root/.local/share/
+        ports:
+          - "9151:9151"
+          - "9101:9101"
+        env:
+          NUCYPHER_KEYRING_PASSWORD: "{{NUCYPHER_KEYRING_PASSWORD}}"
+          NUCYPHER_WORKER_ETH_PASSWORD: "{{NUCYPHER_WORKER_ETH_PASSWORD}}"
+          NUCYPHER_SENTRY_DSN: "{{SENTRY_DSN | default('')}}"
+          NUCYPHER_SENTRY_LOGS: "1"
+
+    - name: "wait a few seconds for the seed node to become available"
+      when: SEED_NODE_URI is not undefined
+      pause:
+        seconds: 15
+
+    - name: "Run Staked Ursula (non-seed)"
       become: yes
       become_user: nucypher
+      when: SEED_NODE_URI is undefined or inventory_hostname != SEED_NODE_URI
       docker_container:
+        recreate: yes
         name: ursula
         state: started
+        pull: yes
+        log_driver: json-file
+        log_options:
+          max-size: 10m
+          max-file: "5"
         image: "{{ nucypher_image | default('nucypher/nucypher:latest') }}"
         restart_policy: "unless-stopped"
-        command: "nucypher ursula run --teacher gemini.nucypher.network:9151 --poa {{nucypher_ursula_run_options | default('')}}"
+        command: "nucypher ursula run {{nucypher_ursula_run_options | default('')}} {{signer_options}} --disable-availability-check --teacher {{SEED_NODE_URI}} --prometheus --metrics-port 9101"
```

Comment:
Perhaps the signer options should be pass at init time. It's not something that will likely be changing between executions